File: app/src/main/java/com/example/todolist/Data/AppDatabase.java
package com.example.todolist.Data;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.example.todolist.models.Note;
import com.example.todolist.models.NoteDao;

@Database(entities = {Note.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract NoteDao employeeDao();
}

File: app/src/main/java/com/example/todolist/Data/AppDatabase.kt
package com.example.todolist.Data

import android.content.Context
import androidx.room.Database
import androidx.room.Room
import androidx.room.RoomDatabase
import com.example.todolist.models.Note

@Database(
    version = 1,
    entities = [Note::class]
)
abstract class AppDatabase : RoomDatabase() {
    abstract fun getNoteDao(): NoteDao

}
